example/voter/src/pkg/ibc.go

```diff
@@ -0,0 +1,37 @@
+package pkg
+
+import (
+	"errors"
+	"math"
+	"strconv"
+	"strings"
+	"unicode"
+)
+
+// ValidateChannelID validates IBC channel ID.
+func ValidateChannelID(channelID string) error {
```

Comment:
shouldn't this check be at execute IBC func level? since it applies to every contract in the same way?

Reply:
I'm not sure about this. This func is being called to check the user input (when a user wants to vote via IBC, he provides the channelID). It is like `sdk.Msg` `ValidateBasic` method.